image-layout.md

```diff
@@ -70,7 +70,7 @@ Those tags will often be represented in an image-layout repository with matching
 ### Example Ref
 
 ```
-$ cat ./refs/v1.0 | jq
+$ cat ./refs/v1.0 | jq .
```

Comment:
@wking  I use jq 1.3.  
ok https://github.com/opencontainers/image-spec/blob/master/HACKING.md#json-schema-formatting  requires for jq 1.5, missing this part.